admin-ui/app/locales/en/translation.json

```diff
@@ -532,14 +532,15 @@
     "issuer": "Issuer",
     "base_endpoint": "Base Endpoint",
     "clean_service_interval": "Clean Service Interval",
+    "clean_batch_chunk_size": "Clean Batch Chunk Size",
     "clean_service_batch_chunk": "Clean Service Batch Chunk",
     "use_local_cache": "Use Local Cache",
     "logging_level": "Logging Level",
     "logging_layout": "Logging Layout",
     "external_logger_configuration": "External Logger Configuration",
     "metric_reporter_interval": "Metric Reporter Interval",
     "metric_reporter_keep_data_days": "Metric Reporter Keep Data Days",
-    "metric_reporter_enabled": "Metric Reporter Enabled",
+    "metric_reporter_enabled": "Enabled Metric Reporter",
```

Comment:
Fix English phrasing: use “Enable Metric Reporter”.

“Enabled Metric Reporter” is awkward in English for a toggle label.

Apply:

```diff
-    "metric_reporter_enabled": "Enabled Metric Reporter",
+    "metric_reporter_enabled": "Enable Metric Reporter",
```